common/types/account_state.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import "math/big"
+
+// AccountState struct represents basic account data: nonce and balance
+// Todo: get rid of big.Int everywhere and replace with uint64
+type AccountState struct {
+	Nonce   uint64   `json:"nonce"`
+	Balance *big.Int `json:"balance"`
+}
+
+// AccountsState is a struct used to dump an entire state root
+type AccountsState struct {
+	Root     string                  `json:"root"`
```

Comment:
it will involve lots of changes to tx processing state which relies on dump results to return a string as the address and not byte[] so fixing this is a broader change.